lib/propolis/src/vmm/mem.rs

```diff
@@ -910,6 +910,12 @@ impl MemCtx {
         let mapping = self.region_covered(region.0, region.1, Prot::READ)?;
         Some(mapping)
     }
+    pub fn read_write_region(
+        &self,
+        region: &GuestRegion,
+    ) -> Option<SubMapping> {
+        self.region_covered(region.0, region.1, Prot::RW)
+    }
```

Comment:
This is dead code (used when I had a more complicated overlay page construct). Will remove.

Reply:
Fixed in 12db242e (though we'll see if it has to come back later).